src/libp2p.rs

```diff
@@ -130,20 +117,6 @@ impl FromStr for PeerId {
     }
 }
 
-pub type Enr = enr::Enr<enr::k256::ecdsa::SigningKey>;
```

Comment:
I don't see a reason to duplicate the `Enr` type in each spec, it can just live here

Reply:
same thing w/ `MessageDomain`

Reply:
Yes, done!